### DISCUSSION TITLE
Changes to nightly build

### DISCUSSION THREAD

**Discussion by Gert6** [AlexandreRouma/SDRPlusPlus#993] [General]
I regularly download the new " versions" on the nightly build platform. Where can I see what is the changes made to the new build.
Thanks.
Gert van Zyl

**Comment by AlexandreRouma** [comment]
Nightlies are built from the master branch so you can just look at the commit history: https://github.com/AlexandreRouma/SDRPlusPlus/commits/master

**Comment by Gert6** [comment]
Thank you very much.Greetings Gert

Sent from Yahoo Mail on Android 
 
  On Sat, Feb 18, 2023 at 13:59, ***@***.***> wrote:   


Nightlies are built from the master branch so you can just look at the commit history: https://github.com/AlexandreRouma/SDRPlusPlus/commits/master
